strip.lua

```diff
@@ -244,7 +244,7 @@ function mine()
         state.position.trunkFacing = 1
     end
 
-    if checkInv() == false and state.mine.index == 1 then
+    if (checkInv() == false and state.mine.index == 1) or (checkFuel() == false) then
```

Comment:
```suggestion
    if ((checkInv() == false or checkFuel()) == false and state.mine.index == 1) then
```

The mine state should not be left unless it hasn't started (index 1) as this state is not easily resumed